packages/opentelemetry-plugin-fetch/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.base",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build"
```

Comment:
You will probably need `skipLibCheck: true` to get the build to pass in CI

Reply:
it was the other issue, I have refactored the tests

Reply:
added